Sources/ArcGISToolkit/MeasureToolbar.swift

```diff
@@ -689,4 +688,19 @@ extension MeasureToolbar: UnitsViewControllerDelegate {
             fatalError("Unsupported unit type")
         }
     }
+    
+    func topViewController(_ controller: UIViewController? = UIApplication.shared.windows.first(where: { $0.isKeyWindow })?.rootViewController) -> UIViewController? {
```

Comment:
Methinks this should be private.

```suggestion
    private func topViewController(_ controller: UIViewController? = UIApplication.shared.windows.first(where: { $0.isKeyWindow })?.rootViewController) -> UIViewController? {
```